plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -1406,8 +1422,8 @@ private String commit(final CommitParams params, URI dockerDaemonUri) throws IOE
         // TODO: add option to pause container
         try (DockerConnection connection = connectionFactory.openConnection(dockerDaemonUri)
                                                             .method("POST")
-                                                            .path("/commit")
-                                                            .query("container", params.getContainer())) {
+                                                            .path("/commit")) {
+            addQueryParamIfNotNull(connection, "container", params.getContainer());
```

Comment:
Commit is mandatory parameter. And it should not be null. Why we need this check?